### PR TITLE
fixed log bug

### PR DESCRIPTION
**Pull request by chrisenuf**

### CODE REVIEW

File: src/fullerite/main.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func initLogrus(ctx *cli.Context) {
 	}
 
 	filename := ctx.String("log_file")
```
Comment (baris) [inline src/fullerite/main.go]: I'd suggest that we get rid of this configuration option too. We know that it's buggy with logrotate.

File: src/fullerite/main.go [hunk 2]
```diff
@@ -95,6 +76,8 @@ func start(ctx *cli.Context) {
 	}
 	initLogrus(ctx)
 	log.Info("Starting fullerite...")
+	log = nil
+	log.Info("Starting fullerite...")
```
Comment (ericchu94) [inline src/fullerite/main.go]: This intentional crash needs to go lol

Comment (chrisenuf) [inline src/fullerite/main.go]: whoops

**Comment by ericchu94** [comment]
@chrisenuf lg2m

**Pull request merged**